### ISSUE TITLE
Lens 6 broke compatibility (again)

### ISSUE THREAD

**Issue by cajual** [kubecost/kubecost-lens-extension#7]
Linked to Zendesk issue 

gz#2388<br><br><i>(related to <a href='https://kubecost.zendesk.com/agent/tickets/2388'>Zendesk ticket #2388</a>)</i>

**Comment by jcharcalla** [comment]
@gibbonsryan Thank you for reporting this issue!

**Comment by Adam-Stack-PM** [comment]
@gibbonsryan, We have this bug prioritized and will review for an upcoming release.

**Comment by sundeepghuman** [comment]
@zorrobyte can you please add some additional context here so I can assign this to the appropriate Engineer? I am not familiar with Lens 6. Thanks!

**Comment by dwbrown2** [comment]
@zorrobyte any updates? 

@nealormsbee you mentioned it might be related to the `localhost:9090` option?

**Comment by AjayTripathy** [comment]
Assigned to @linhlam-kc after speaking with @robfaraj . Removed p1 note; @linhlam-kc will own priority etc. here, not necessarily core engineering.

**Comment by nealormsbee** [comment]
>@nealormsbee you mentioned it might be related to the localhost:9090 option?

Fairly confident in this, yes. Today, if a UI is set to target a `localhost` backend, it will drop whatever port is configured and use `localhost:9090`. @michaelmdresser has a related issue here:

https://github.com/kubecost/cost-analyzer-frontend/issues/1015

And a PR that I believe resolves both of these issues: https://github.com/kubecost/cost-analyzer-frontend/pull/1029

As noted there, we'll want to test pretty carefully in different scenarios, because this has been the behavior for a long time.

**Comment by dwbrown2** [comment]
Thanks, Neal! re: https://github.com/kubecost/cost-analyzer-frontend/pull/1029 -- aiming for 1.97 or 1.98?

cc @Adam-Stack-PM

**Comment by linhlam-kc** [comment]
Update: I submitted new PR to fix the incompatibility issue. The issue is similar to this one https://github.com/lensapp/lens/issues/3404 which  Lens extension engine range ^5.0.0 not working  with Lens 6.

**Comment by dwbrown2** [comment]
Awesome, @linhlam-kc! Good to close this or still waiting on confirmation of fix?

**Comment by linhlam-kc** [comment]
@dwbrown2 Let's wait for this PR to be reviewed and merged https://github.com/kubecost/kubecost-lens-extension/pull/8 . It is working from my local dev but it is better to have 2nd eyes on it before merging.

**Comment by linhlam-kc** [comment]
v0.0.4 is released for supporting customers using Lens 6+ https://github.com/kubecost/kubecost-lens-extension/releases/tag/v0.0.4
